Makefile

```diff
@@ -61,6 +61,8 @@ test: unit_test
 
 build: cdc
 
+build-failpoint: failpoint-enable cdc failpoint-disable
```

Comment:
Are they executed in serial when `make -j4`?

Reply:
nice question...